Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1105,13 +1105,16 @@ class BrowserViewController: UIViewController {
             updateInContentHomePanel(url as URL)
         }
     }
-
+    var rewardsXHRLoadURL: URL?
```

Comment:
Can we add a comment about what this is for and also add a line of whitespace below it